configs/projects/_shared-agent-settings.rb

```diff
@@ -196,7 +195,7 @@
   'no-dtls1',
   'no-idea',
   'no-seed',
-  'no-ssl2-method',
+  # 'no-ssl2-method',
```

Comment:
Is this option deprecated in 1.1.0?

Reply:
yeah, I forgot to move this in to ssl version specific settings

Reply:
Updated.